### PR TITLE
But can I have it for dinner?

### PR DESCRIPTION
**Pull request by ccocks**
I finally found a repository that makes it easy to earn the Pull Shark badge. 😆

### CODE REVIEW

**Comment by ccocks** [comment]
Thanks in advance if you do merge this. My profile page is looking bland.

File: index.js [hunk 1]
```diff
@@ -46,3 +46,12 @@ module.exports.butWhyIsCinnamonToastCrunchGood =
     return "It's simply a fact of life, you know? Birds fly, gravity falls, " +
       "stevens universe, Cinnamon Toast Crunch is good."
   }
+
+module.exports.butCanIHaveItForDinner = 
+  function nobodyToldMeYouThatYouCould() {
+
+return "Haven't heard of it, but you can have it for breakfast... and lunch..." +
```
Comment (towerofnix) [inline index.js]: There should be a space at the end of this line (inside the quotation marks).

**Review by towerofnix — CHANGES_REQUESTED** [review]
Beautiful! Thank you for sharing this newfound knowledge. Please add the missing space and update the code so that it's indented the same way as the other lines in the file (no need for blank lines inside the function either). Then it's perfect!

**Comment by ccocks** [comment]
Can you merge this now?

**Comment by towerofnix** [comment]
Thanks for getting the space fixed up, but sorry, you'll still need to fix the indentation and remove the blank lines! Here is your current code:

```js

module.exports.butCanIHaveItForDinner = 
  function nobodyToldMeYouThatYouCould() {

return "Haven't heard of it, but you can have it for breakfast... and lunch... " + 
  "and as a snack... and pretty much whenever you want... " +
  "I guess that includes dinner too."

}
```

Here is what it should be:

```js
module.exports.butCanIHaveItForDinner = 
  function nobodyToldMeYouThatYouCould() {
    return "Haven't heard of it, but you can have it for breakfast... and lunch... " + 
      "and as a snack... and pretty much whenever you want... " +
      "I guess that includes dinner too."
  }
```

**Comment by ccocks** [comment]
Oh ok.

**Comment by ccocks** [comment]
Sorry about that

**Review by towerofnix — APPROVED** [review]
Thank you! We'll have this published on NPM as why-do-kids-love-the-taste-of-cinnamon-toast-crunch@3.3.0. 🎉

**Comment by ccocks** [comment]
👍🏼

**Pull request merged**